conda/exceptions.py

```diff
@@ -159,7 +159,32 @@ def __init__(self, target_path, incompatible_package_dists, context):
 
 class CommandNotFoundError(CondaError):
     def __init__(self, command):
-        message = "Conda could not find the command: '%(command)s'"
+        build_commands = {
+            'build',
```

Comment:
@kalefranz why not reuse the build commands dict here https://github.com/conda/conda/pull/4963/files#diff-aa5c1e6732e4802903e91fe909341fd0R19